components/raftstore/src/store/peer.rs

```diff
@@ -1796,12 +1801,14 @@ impl Peer {
             ConfChangeType::AddLearnerNode => {
                 return Ok(());
             }
-            ConfChangeType::BeginMembershipChange | ConfChangeType::FinalizeMembershipChange => {
-                unimplemented!()
-            }
         }
         let healthy = self.count_healthy_node(progress.voters());
-        let quorum_after_change = raft::majority(progress.voter_ids().len());
+        let quorum_after_change = match ctx.cfg.quorum_algorithm {
```

Comment:
Maybe we can use the function pointer like raft-rs?

Reply:
I would like to but the function pointer in `Raft` is private.

Reply:
Got it.